metis-common/metis-common-utils/src/main/java/eu/europeana/metis/utils/CompressedFileHandler.java

```diff
@@ -204,6 +208,10 @@ private static void extractTarFile(final Path compressedFile, final Path destina
     }
   }
 
+  private static String replaceBannedCharacters(String entryName) {
+    return entryName.replaceAll("[" + FILE_NAME_BANNED_CHARACTERS + "]", "+");
```

Comment:
my suggestion would be to use underscore `_` instead of plus `return entryName.replaceAll("[" + FILE_NAME_BANNED_CHARACTERS + "]", "_");`

Reply:
Done